src/index.js

```diff
@@ -118,67 +118,61 @@ class MyArray {
     }
     return resultMassive;
   }
+
   toString() {
     let str = '';
 
     for (let i = 0; i < this.length; i++) {
-      if (i === (this.length - 1)) {
-        str += `${this[i]}`;
-      } else {
-        str += `${this[i]},`;
-      }
+      i === (this.length - 1) ? str += `${this[i]}` : str += `${this[i]},`;
     }
     return str;
   }
 
   slice(begin, end) {
-    const instanceMasive = this;
-    const newMasive = new MyArray();
+    const newMassif = new MyArray();
     let startValueIndex = begin ? begin : 0;
-    let endValueIndex = end ? end : instanceMasive.length;
+    let endValueIndex = end ? end : this.length;
 
-    startValueIndex = begin < 0 ? instanceMasive.length + begin : startValueIndex;
-    endValueIndex = end < 0 ? instanceMasive.length + end : endValueIndex;
+    startValueIndex = begin < 0 ? this.length + begin : startValueIndex;
+    endValueIndex = end < 0 ? this.length + end : endValueIndex;
 
     for (let i = startValueIndex; i < endValueIndex; i++) {
-      newMasive.push(instanceMasive[i]);
+      newMassif.push(this[i]);
     }
-    return newMasive;
+    return newMassif;
   }
 
   sort(callback) {
-    const mas = this;
-    let max = null;
+    const massif = this;
+    let maxValue = null;
 
     if (callback) {
-      for (let i = 0; i < mas.length - 1; i++) {
-        for (let j = 0; j < mas.length - 1; j++) {
-          if (callback(mas[j], mas[j + 1]) > 0) {
-            max = mas[j];
-            mas[j] = mas[j + 1];
-            mas[j + 1] = max;
+      for (let i = 0; i < massif.length - 1; i++) {
+        for (let j = 0; j < massif.length - 1; j++) {
+          if (callback(massif[j], massif[j + 1]) > 0) {
+            maxValue = massif[j];
+            massif[j] = massif[j + 1];
+            massif[j + 1] = maxValue;
           }
         }
       }
     } else {
-      for (let i = 0; i < mas.length - 1; i++) {
-        for (let j = 0; j < mas.length - i; j++) {
-          if (String(mas[j]) > String(mas[j + 1])) {
-            max = mas[j];
-            mas[j] = mas[j + 1];
-            mas[j + 1] = max;
+      for (let i = 0; i < massif.length - 1; i++) {
+        for (let j = 0; j < massif.length - 1; j++) {
+          if (String(massif[j]) > String(massif[j + 1])) {
```

Comment:
if you want, you can cut your code :        if (`massif[j]` > `massif[j + 1]`) {